torch/testing/_internal/common_nn.py

```diff
@@ -596,6 +596,19 @@ def l1loss_no_reduce_test():
         pickle=False)
 
 
+def l1loss_no_reduce_complex_test():
```

Comment:
how is this different from `test_pointwise_loss_broadcast` for the case when reduction is set to `None`?

Reply:
I think it also invokes the cpp api and tests cpp api parity